core/src/main/java/org/kohsuke/stapler/IndexHtmlDispatcher.java

```diff
@@ -36,7 +36,7 @@ public String toString() {
      * Returns a {@link IndexHtmlDispatcher} if and only if the said class has {@code index.html} as a side-file
      */
     static Dispatcher make(ServletContext context, Class c) {
```

Comment:
Do you mean this method returning null? That should be a valid scenario in case no index.html is found.